src/plugins/discover/public/application/main/utils/fetch_all.ts

```diff
@@ -72,12 +90,13 @@ export function fetchAll(
   const sendErrorTo = (
     ...errorSubjects: Array<DataMain$ | DataDocuments$ | DataTotalHits$ | DataCharts$>
   ) => {
-    return (error: Error) => {
+    return (error: RequestError) => {
       if (error instanceof Error && error.name === 'AbortError') {
         return;
       }
 
-      data.search.showError(error);
+      const debounceKey = error.message + error.stack + error?.body;
```

Comment:
appending error.body would, since it's an object lead to an `[object Object]` in the string, so it should be stringified